pymongo/asynchronous/mongo_client.py

```diff
@@ -1140,13 +1257,16 @@ def option_repr(option: str, value: Any) -> str:
             return f"{option}={value!r}"
 
         # Host first...
-        options = [
-            "host=%r"
-            % [
-                "%s:%d" % (host, port) if port is not None else host
-                for host, port in self._topology_settings.seeds
+        if self._topology is None:
+            options = self._resolve_srv_info["seeds"]
```

Comment:
Do we have tests for this? If not could we add some. I'd like to see the behavior difference in repr().